SPECS/python-libclang/python-libclang.spec

```diff
@@ -0,0 +1,41 @@
+%global pypi_name libclang
+
+Summary:        Clang's python bindings
+Name:           python-%{pypi_name}
+Version:        14.0.6
+Release:        2%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/sighingnow/libclang
+Source0:        %{pypi_source}
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildArch:      noarch
+
+%description
+This library makes it easier to install clang's python bindings.
+
+%package -n     python3-%{pypi_name}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+Summary:        %{summary}
+
+%description -n python3-%{pypi_name}
+This library makes it easier to install clang's python bindings.
+
+%prep
+%autosetup -n %{pypi_name}-%{version}
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n python3-%{pypi_name}
+%license LICENSE.TXT
+%{python3_sitelib}/*
+
+%changelog
+* Mon Oct 17 2022 Riken Maharjan <rmaharjan@microsoft.com> - 0.5.3-2
```

Comment:
Not sure why release is 2

Reply:
I totally missed this while creating the PR. I made this change while ago. 
Fixed the mistake now.